tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -11,9 +11,15 @@
 #include <bs_pc_backchannel.h>
 #include <time_machine.h>
 
+#if defined CONFIG_BT_MESH_USES_TINYCRYPT
```

Comment:
I don't think we need to support tests for both. Otherwise we need to compile 2 binaries for each crypto library.